mix.exs

```diff
@@ -73,25 +73,25 @@ defmodule Membrane.RTP.Plugin.MixProject do
 
   defp deps do
     [
-      {:membrane_core, "~> 0.9.0", override: true},
+      {:membrane_core, "~> 0.9.0"},
       {:membrane_rtp_format, "~> 0.3.1"},
-      {:membrane_rtp_h264_plugin, "~> 0.8.0", only: :test},
-      {:membrane_rtp_mpegaudio_plugin, "~> 0.7.0", only: :test},
-      {:membrane_h264_ffmpeg_plugin, "~> 0.16.0", only: :test},
-      {:membrane_element_pcap,
-       github: "membraneframework/membrane-element-pcap", tag: "v0.4.0", only: :test},
-      {:membrane_element_udp, "~> 0.6.0", only: :test},
-      {:membrane_hackney_plugin, "~> 0.6.0", only: :test},
       {:ex_libsrtp, "~> 0.3.0", optional: true},
       {:qex, "~> 0.5.1"},
       {:bunch, "~> 1.0"},
       {:heap, "~> 2.0.2"},
       {:bimap, "~> 1.1.0"},
 
+      # Test
+      {:membrane_rtp_h264_plugin, "~> 0.9.0", only: :test},
+      {:membrane_rtp_mpegaudio_plugin, "~> 0.8.0", only: :test},
+      {:membrane_h264_ffmpeg_plugin, "~> 0.17.0", only: :test},
+      {:membrane_pcap_plugin,
+       github: "membraneframework/membrane_pcap_plugin", tag: "v0.5.0", only: :test},
+      {:membrane_hackney_plugin, "~> 0.7.0", only: :test},
+
       # Dev
       {:ex_doc, "~> 0.21", only: :dev, runtime: false},
       {:dialyxir, "~> 1.0", only: :dev, runtime: false},
-      {:excoveralls, ">= 0.8.0", only: :test},
```

Comment:
If you're removing this, remove the leftovers in `project` callback

Reply:
you can also apply the usual stuff to `docs`, i.e. `LICENSE: [title: License]` and `formatters: ["HTML"],`